plugins/modules/datadog_monitor.py

```diff
@@ -167,6 +167,28 @@
           - Integer from 1 (high) to 5 (low) indicating alert severity.
         type: int
         version_added: 4.6.0
+    notification_preset_name:
+        description:
+          - Toggles the display of additional content sent in the monitor notification.
+        choices:
+            - show_all
+            - hide_query
+            - hide_handles
+            - hide_all
+        type: str
+        version_added: 7.2.0
+    renotify_occurrences:
+        description:
+            - The number of times re-notification messages should be sent on the current status at the provided re-notification interval.
+        type: int
+        version_added: 7.2.0
+    renotify_statuses:
+        description:
+            - The types of monitor statuses for which re-notification messages are sent. Valid values are alert, warn, no data.
```

Comment:
Are these the extract strings that can be passed in? Then they should be formatted as values:
```suggestion
            - The types of monitor statuses for which re-notification messages are sent. Valid values are C(alert), C(warn), C(no data).
```
Also if these are the *only* values that can be passed in, you probably want to use `choices:` to limit the values to this option to exactly these values.